CHANGELOG.md

```diff
@@ -9,8 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Update to `dusk-plonk` `0.9` [#62](https://github.com/dusk-network/schnorr/issues/62)
 - Update `dusk-poseidon` from `0.21` to `0.22.0-rc` [#59](https://github.com/dusk-network/schnorr/issues/59)
 - Update `dusk-pki` from `0.7` to `0.8.0-rc` [#59](https://github.com/dusk-network/schnorr/issues/59)
+
```

Comment:
The changelog is misleading. Looks like you modified also `dusk-poseidon` and dusk-pki` but without update them in the changelog, but just adding `dusk-plonk` 0.9.

I assume those are coming from a previous `rc` (0.8). That means we should release the 0.8 (from 0.8-rc make 0.8) and publish it on crates.io (with the relevant changelog modification) before create a new rc (0.9).

This should be the way to go, but I understand it's more time consuming.

As alternative, we could just modifying the changelog; adding a version entry `0.8-rc` with all the changes related move to it, and then having under `unreleased` the changes related to `0.9-rc`. 

Since the `0.8-rc.0` is published on crates.io this would be enough. It will correctly report the history of the changes and highlights that 0.8 was never released.